homework/english/viacheslav_stashchuk/lecture1/task1_grammar.md

```diff
@@ -0,0 +1,45 @@
+#Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+Your mum (work) today? - Is your mum working today? 
+Why this boys (to talk) now? - Why are these boys talking now?
+Where he (to live)? - Where does he live?
+Their mother (cook) very well? - Is their mother cooking very well?
+Lili is not rich, she (not to drive) a Mercedes. - Lili is not rich, she does not drive a Mercedes.
+Let´s go out. It (not rain) now. - Let´s go out. It is not raining now.
+I have to go now. It (get) dark - I have to go now. It is getting dark
+A young man usually (to stand) at the window. - A young man usually stands at the window.
+Ann usually (make).......... doll dresses herself. Ann usually makes doll dresses herself.
+His wife (have) coffee at the moment? - Is his wife having coffee at the moment?
+Let’s go for a walk. It (not/rain). - Let’s go for a walk. It is not raining.
+Look! The boys (come) home. - Look! The boys are coming home.
+Maila (like) Chinese food? - Does Maila like Chinese food?
+My grandmother (not to work). She is on pension. - My grandmother does not work. She is on pension.
+What your brother (watch) now? - What is your brother watching now?
+Where you (live)? - Where do you live?
+You (to play) the piano well? - Do you play the piano well?
+Where their brother (swim) now? - Where is their brother swimming now?
+What they (to eat) at breakfast? - What do they eat at breakfast?
+They (to swim) at the moment? - Are they swimming at the moment?
+Mike (not wear) a tie today. - Mike is not wearing a tie today.
+Maila (not to watch) TV every morning. - Maila does not watch TV every morning.
+Listen! Blanca (sing) in the bathroom. - Listen! Blanca is singing in the bathroom.
+Bea sometimes (read) comics. - Bea sometimes reads comics.
+
+#Найдите ошибки и исправьте их.
+We not going to school today. - We are not going to school today.
+What you doing after school? - What are you doing after school?
+At the moment Peter is work in Russia. - At the moment Peter is working in Russia.
+Does he got a new car? - Has he got a new car?  
```

Comment:
this is also correct, but for the past simple
Does he **get** a new car?

Reply:
It is correct. When we are talking about possession, relationships, illnesses and characteristics of people or things we can use either have or have got. The have got forms are more common in an informal style. Have got has the same meaning as have and both are used as present tenses. Note that have got is not the present perfect of get